### ISSUE TITLE
I have not been able to reproduce your results.

### ISSUE THREAD

**Issue by yyf17** [krantiparida/beyond-image-to-depth#4]
thank you for your excellent work. There is no training code in the Git project, only the test code. So I wrote a trainer myself. But I have not been able to reproduce your results. I used my own training checkpoint during the test. The test command used is:
```
python test.py \
--gpu_ids 0 \
--dataset mp3d \
--img_path data/echoes_obs/mp3d/scene_observations_128.pkl \
--audio_path data/echoes_navigable \
--checkpoints_dir data/models/mp3d_128 \
--audio_sampling_rate 16000 \
--max_depth 10.0
```
The final result is:
![image](https://user-images.githubusercontent.com/34536352/123534886-e8ebf500-d752-11eb-9851-7ab8cb198be8.png)

Can you give me some suggestions? Thanks a lot.

**Comment by krantiparida** [comment]
Hi, did you try using the pre-trained model provided?

**Comment by yyf17** [comment]
Yes.The results are as following:
![image](https://user-images.githubusercontent.com/34536352/123545606-764b3b80-d78b-11eb-9b06-2ade9310b2b4.png)

**Comment by krantiparida** [comment]
I think the issue is with training parameters then. Try to use the exact parameter's value mentioned in the paper and evaluate the result using the best model in the validation set.

**Comment by krantiparida** [comment]
@yyf17 can you please run the training code available and see how are the results?

**Comment by yyf17** [comment]
@krantiparida OK,I will.

**Comment by catherine-qian** [comment]
Thanks for the work!
I would also like to use the training script. Could you please let me know what should I specify for 'metadatapath path_to_metadata'?

**Comment by krantiparida** [comment]
`metadatapath` conatins the scene wise splits for train, val and test. The content is available inside the [dataset](https://github.com/krantiparida/beyond-image-to-depth/tree/main/dataset/metadata/mp3d) folder in this repo.
